optuna_dashboard/ts/components/AppDrawer.tsx

```diff
@@ -246,6 +248,24 @@ export const AppDrawer: FC<{
                 <ListItemText primary="Analytics" sx={styleListItemText} />
               </ListItemButton>
             </ListItem>
+            {studyDetail?.is_preferential && (
```

Comment:
How about checking `studySummary?.is_preferential` as well? Perhaps it is better to define the function like below.

https://github.com/optuna/optuna-dashboard/blob/238dffa80f347081fb93b77436c9c21dd4153439/optuna_dashboard/ts/state.ts#L82-L88